src/components/Board.tsx

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import { Grid, Paper } from "@mui/material";
+
+const formatNumber: (num: number) => string = (num: number) =>
```

Comment:
should be in different file i.e., helper file

Reply:
completed